Sort Pod environment values in alphabetical order

stevejr opened the pull request:
Fixes: #739

Signed-off-by: Steve Richards <srichards@mirantis.com>

stevejr committed:
Sort Pod environment values in alphabetical order

Fixes: #739

Signed-off-by: Steve Richards <srichards@mirantis.com>

Nokel81 commented on src/renderer/components/+workloads-pods/pod-container-env.tsx:
```diff
@@ -11,6 +11,7 @@ import { secretsStore } from "../+config-secrets/secrets.store";
 import { configMapsStore } from "../+config-maps/config-maps.store";
 import { Icon } from "../icon";
 import { base64, cssNames } from "../../utils";
+import _ from "lodash";
```
On line 4 above we already import from lodash, could you change all the uses of `flatten` to `_.flatten` and remove the above?

stevejr replied:
Done

jim-docker replied:
Or couldn't you just import `sortBy` like `flatten`? Seeing `_` is so 2010 :)

Nokel81 commented on src/renderer/components/+workloads-pods/pod-container-env.tsx:
```diff
@@ -40,7 +41,9 @@ export const ContainerEnvironment = observer((props: Props) => {
 
   const renderEnv = () => {
-    return env.map(variable => {
+    let orderedEnv = _.sortBy(env, 'name');
```
nit: use `const`

stevejr replied:
Done.

stevejr committed:
Consolidated to a single lodash import. Corrected Lint failure

- Removed the additional import of flatten and updated as needed;
- Changed the variabled `orderedEnv` to be a const;

Signed-off-by: Steve Richards <srichards@mirantis.com>
